Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift

```diff
@@ -36,15 +36,33 @@ final class SavedRecipesTableViewCell: UITableViewCell {
         configureLayout()
     }
 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+
+        contentView.frame = contentView.frame.inset(by: UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10))
+    }
+
     // MARK: Configure
 
-    func update(with viewModel: ViewModel) {
+    func update(with viewModel: SavedRecipe) {
```

Comment:
Please revert this change. 

Now the cell is depend on the `SavedRecipe` which is not correct!